platforms/jvm/plugins-application/src/integTest/groovy/org/gradle/java/JavaApplicationOutgoingVariantsIntegrationTest.groovy

```diff
@@ -29,49 +31,44 @@ class JavaApplicationOutgoingVariantsIntegrationTest extends AbstractIntegration
         repo.module("test", "runtime-only", "1.0").publish()
 
         createDirs("other-java", "java", "consumer")
-        settingsFile << "include 'other-java', 'java', 'consumer'"
-        buildFile << """
-def artifactType = Attribute.of('artifactType', String)
-
-allprojects {
-    repositories { maven { url = '${mavenRepo.uri}' } }
-}
-
-project(':other-java') {
-    apply plugin: 'java-library'
-}
-
-project(':java') {
-    apply plugin: 'application'
-    dependencies {
-        implementation project(':other-java')
-        implementation files('file-dep.jar')
-        compileOnly 'test:compile-only:1.0'
-        implementation 'test:implementation:1.0'
-        runtimeOnly 'test:runtime-only:1.0'
-    }
-}
-
-project(':consumer') {
-    configurations { consume }
```

Comment:
Creating configuration here triggers `consume` property lookup in Groovy, what is an IP violation. It seems like it's unavoidable now, and this is how `NamedDomainObjectContainer` works?
@gradle/bt-jvm

Reply:
Not sure about groovy behavior but you can just use create("consume") instead to avoid that behavior. The syntax used there is not part of what the test is verifying

Reply:
Yeah, for real. Did that